Adapt "--range" error text for node 0.55.0

https://rest.fs.neo.org/Aku38wpn2xEqfEPxmwyAa7N3VnAkMV8PEVkpLqwKM1z3/5710-1785335656/index.html#suites/ac3a91d49812d43ae62588fd8167fc9d/a2aa9686e8cc5f7a/
https://rest.fs.neo.org/Aku38wpn2xEqfEPxmwyAa7N3VnAkMV8PEVkpLqwKM1z3/5716-1785364478/index.html
https://rest.fs.neo.org/Aku38wpn2xEqfEPxmwyAa7N3VnAkMV8PEVkpLqwKM1z3/5716-1785361320/index.html

All fail with
```
AssertionError: Regex pattern did not match.
  Expected regex: '--binary cannot be used with --range'
  Actual message: "Command: ./neofs-cli --config /Users/runner/work/neofs-node/neofs-node/neofs-testcases/wallet_config.yml object get --rpc-endpoint 'localhost:54203' --wallet '/Users/runner/work/neofs-node/neofs-node/neofs-testcases/test-run-2026-07-29-20-08-29-810469/wallet-0b19dd86-f370-4fc6-ae35-37c50c783f8c.json' --cid '76QfoqcvdYSwX5BD6VFaBumD9sx87XAmh3PiMX9bERv8' --oid 'BTZbTAesDAKPogpCY27dLBGW7eTVVufEvqfWKztBeZv' --file '/Users/runner/work/neofs-node/neofs-node/neofs-testcases/test-run-2026-07-29-20-08-29-810469/TestObjectsDir/74af0799-3631-4b33-b21d-91dc9b89523c' --no-progress --binary --range '0:1'\nError:\nreturn code: 1\noutput: Error: --binary cannot be used with a range\n"
```
Which is annoying. Related to https://github.com/nspcc-dev/neofs-node/pull/4094 change.

--- a/pytest_tests/tests/object/test_object_ranged_get.py
+++ b/pytest_tests/tests/object/test_object_ranged_get.py
@@ -334,14 +334,16 @@ class TestObjectRangedGet(TestNeofsBase):
 
     @allure.title("Ranged GET rejects --binary together with --range")
     @pytest.mark.simple
-    def test_ranged_get_binary_rejected(self, default_wallet: NodeWallet, default_container: str):
+    def test_ranged_get_binary_rejected(
+        self, skip_if_extended_range_unsupported, default_wallet: NodeWallet, default_container: str
+    ):
         file_size = self.neofs_env.get_object_size("simple_object_size")
         file_path = generate_file(file_size)
         oid = _put_object(self.neofs_env, default_wallet, default_container, file_path)
 
         cli = NeofsCli(self.neofs_env.shell, NEOFS_CLI_EXEC, WALLET_CONFIG)
         out_file = os.path.join(get_assets_dir_path(), TEST_OBJECTS_DIR, str(uuid.uuid4()))
-        with pytest.raises(Exception, match=r"--binary cannot be used with --range"):
+        with pytest.raises(Exception, match=BINARY_WITH_RANGE):
             cli.object.get(
                 rpc_endpoint=self.neofs_env.sn_rpc,
                 wallet=default_wallet.path,

